src/app/components/flow-chart-editor/flow-chart-editor.component.ts

```diff
@@ -145,7 +148,7 @@ export class FlowChartEditorComponent {
         x2: edge.line.endPoint.x, y2: edge.line.endPoint.y,
         selected: this.selectionInModel.isEdgeHighlightedInDrawing(edge.getKey(), this.layoutModel!),
         arrow: edge.isLastSegment,
-        isError: edge.optionalOriginalText === 'error'
+        isError: FlowChartEditorComponent.errorForwardNames.includes(edge.optionalOriginalText||'success') || edge.getFrom().originalStyle === 'errorOutline'
       }})
```

Comment:
De berekening van de styles en isError velden is complex geworden. Ik zou graag zien dat dit ergens in een Karma test wordt getest.